src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -60,6 +61,14 @@ public virtual ICommandExecutive CommandExecutive
         }
 
         /// <summary>
+        /// Return logger for sending information to Dynamo console
+        /// </summary>
+        public ILogger Logger
+        {
+            get { return dynamoModel.Logger; }
```

Comment:
@ikeough were you proposing something like this, or another more conservative logging access for extensions?